src/zenml/zen_stores/schemas/tag_schemas.py

```diff
@@ -138,7 +138,7 @@ def from_request(
             The converted schema.
         """
         return cls(
-            id=request.tag_resource_id,
+            id=uuid4(),
```

Comment:
```suggestion
```

You don't really need to generate a UUID here anymore, there's a default factory set in the base class.